bin/cheribsdtest/cheribsdtest_bounds_stack.c

```diff
@@ -192,274 +187,240 @@ test_bounds_stack_dynamic_cap(const struct cheri_test *ctp __unused)
 	uint8_t * __capability arrayp = (__cheri_tocap uint8_t * __capability)&array[0];
-
 	test_bounds_precise(arrayp, sizeof(array));
 }
 
-void
-test_bounds_stack_dynamic_4096(const struct cheri_test *ctp __unused)
+CHERIBSDTEST(test_bounds_stack_dynamic_4096,
+    "Check bounds on a 4,096-byte dynamic stack allocation")
 {
-
 	test_bounds_stack_alloca(4096);
 }
 
-void
-test_bounds_stack_static_8192(const struct cheri_test *ctp __unused)
+CHERIBSDTEST(test_bounds_stack_static_8192,
+    "Check bounds on a 8,192-byte static stack allocation")
 {
 	uint8_t array[8192];
 	uint8_t * __capability arrayp = (__cheri_tocap uint8_t * __capability)&array[0];
-
 	test_bounds_precise(arrayp, sizeof(array));
 }
 
-void
-test_bounds_stack_dynamic_8192(const struct cheri_test *ctp __unused)
+CHERIBSDTEST(test_bounds_stack_dynamic_8192,
+    "Check bounds on a 8,192-byte dynamic stack allocation")
 {
-
 	test_bounds_stack_alloca(8192);
 }
 
-void
-test_bounds_stack_static_16384(const struct cheri_test *ctp __unused)
+CHERIBSDTEST(test_bounds_stack_static_16384,
+    "Check bounds on a 16,384-byte static stack allocation")
 {
 	uint8_t array[16384];
 	uint8_t * __capability arrayp = (__cheri_tocap uint8_t * __capability)&array[0];
-
 	test_bounds_precise(arrayp, sizeof(array));
 }
 
-void
-test_bounds_stack_dynamic_16384(const struct cheri_test *ctp __unused)
+CHERIBSDTEST(test_bounds_stack_dynamic_16384,
+    "Check bounds on a 16,384-byte dynamic stack allocation")
 {
-
 	test_bounds_stack_alloca(16384);
 }
 
-void
-test_bounds_stack_static_32768(const struct cheri_test *ctp __unused)
+CHERIBSDTEST(test_bounds_stack_static_32768,
+    "Check bounds on a 32,768-byte static stack allocation")
 {
 	uint8_t array[32768];
 	uint8_t * __capability arrayp = (__cheri_tocap uint8_t * __capability)&array[0];
-
 	test_bounds_precise(arrayp, sizeof(array));
 }
 
-void
-test_bounds_stack_dynamic_32768(const struct cheri_test *ctp __unused)
+CHERIBSDTEST(test_bounds_stack_dynamic_32768,
+    "Check bounds on a 32,768-byte dynamic stack allocation")
 {
-
 	test_bounds_stack_alloca(32768);
 }
 
-void
-test_bounds_stack_static_65536(const struct cheri_test *ctp __unused)
+CHERIBSDTEST(test_bounds_stack_static_65536,
+    "Check bounds on a 65,536-byte static stack allocation")
 {
 	uint8_t array[65536];
 	uint8_t * __capability arrayp = (__cheri_tocap uint8_t * __capability)&array[0];
-
 	test_bounds_precise(arrayp, sizeof(array));
 }
 
-void
-test_bounds_stack_dynamic_65536(const struct cheri_test *ctp __unused)
+CHERIBSDTEST(test_bounds_stack_dynamic_65536,
+    "Check bounds on a 65,536-byte dynamic stack allocation")
 {
-
 	test_bounds_stack_alloca(65536);
 }
 
-void
-test_bounds_stack_static_131072(const struct cheri_test *ctp __unused)
+CHERIBSDTEST(test_bounds_stack_static_131072,
+    "Check bounds on a 131,072-byte static stack allocation")
 {
 	uint8_t array[131072];
 	uint8_t * __capability arrayp = (__cheri_tocap uint8_t * __capability)&array[0];
-
```

Comment:
FWIW, I do find it odd to drop the blank line between variable declarations and code, but that's minor.

Reply:
oops, I botched a vi command and and deleted a bunch of lines without an easy, automatic way to recover in this file.  I'll re-review that diff.